openfisca_core/entities.py

```diff
@@ -1,5 +1,7 @@
 # -*- coding: utf-8 -*-
 
+import traceback
+
```

Comment:
https://www.python.org/dev/peps/pep-0008/#imports

> Imports should be grouped in the following order:
> 
> * standard library imports
> * related third party imports
> * local application/library specific imports
>
> You should put a blank line between each group of imports.

This rule should not be observed for openfisca-core ?